scripts/uberenv/spack_configs/blueos_3_ppc64le_ib/packages.yaml

```diff
@@ -116,6 +116,8 @@ packages:
     variants: ~shared
   mfem:
     variants: ~mpi~metis~zlib
+  umpire:
+    variants: ~shared
```

Comment:
So, does this mean that when we build TPLs through spack, that Axom requires static compilation for conduit and umpire?  If that's so, why do we do this?

Reply:
Anything in the packages.yaml is specific to us and the SYS_TYPE in the path.  We do it this way so it is not hardcoded in the shared axom/package.py which goes up to Spack proper.